_includes/footer.html

```diff
@@ -6,7 +6,7 @@
           <ul class="contact-list">
             <li class="p-name">
               {%- if site.author -%}
-                <a class="black-link" href="{{ site.url }}/about.html">
+                <a class="black-link" href="{{ site.url | append: '/about.html' }}">
```

Comment:
You can change `<a class="black-link" href="{{ site.url }}/about.html">` to  `<a class="black-link" href="{{ site.url }}about.html">`.

Reply:
> You can change `<a class="black-link" href="{{ site.url }}/about.html">` to `<a class="black-link" href="{{ site.url }}about.html">`.

Just remove a extra slash.

Reply:
The append takes care of `site.url` being either `example.com` or `example.com/` which is nice. The output will always be `example.com/about.html`.